src/components/addToPlaylistButton/AddToPlaylistButton.tsx

```diff
@@ -40,6 +42,16 @@ export const AddToPlaylistButton = ({ videoId }: Props) => {
 
   const { data: playlists } = useOwnPlaylistsQuery();
 
+  const playlistCreatedHotjarEvent = () => {
```

Comment:
Do you think it's nicer for the HotJar service to wrap these calls? E.g `HotjarFactory.hotjar().videoAddedToPlaylist()` interesting to hear thoughts.

Also I think it might be more "react" to use a hook to get the hotjar instance rather than a factory. So something like

```
// a hooks file
const useHotjar = () => HotjarFactory.hotjar();

// in component
const hotjar = useHotjar();

hotjar.videoAddedToPlaylist();
```
But maybe that's a bit overkill

Reply:
I would keep a more general approach sending simple events instead of creating a wrapper method for each particular event.
It should give us more flexibility in sending events depending on the page the button is used.